packages/credential-ld/src/suites/Ed25519Signature2020.ts

```diff
@@ -48,7 +48,7 @@ export class VeramoEd25519Signature2020 extends VeramoLdSignature {
     const verificationKey = new Ed25519VerificationKey2020({
       id,
       controller,
-      publicKeyMultibase: this.preSigningKeyModification(u8a.fromString(key.publicKeyHex, 'hex')),
+      publicKeyMultibase: u8a.toString(u8a.fromString(key.publicKeyHex, 'hex'), 'base58btc'),
```

Comment:
I'm not sure that's enough for it to be a multibase public key representation.
`publicKeyMultibase` is supposed to be self descriptive, meaning that the key type (Ed25519) should be determined by a prefix.
The internal `publicKeyHex` is supposed to be the raw public key data, without such a prefix.
The encoding you are doing here seems to be more suitable for `publicKeyBase58` rather than `publicKeyMultibase`.

If you're having trouble matching a Verification Method from a DID document against the locally managed key (#1139), it's more likely that there's a bug in `extractPublicKeyBytes`. Perhaps try to check if [this line](https://github.com/cheqd/veramo/blob/027fb0a550d6089f8731dd9e6b2084c6294a9bd5/packages/utils/src/did-utils.ts#L141) is doing the right thing.